mathesar_ui/src/pages/schema/SchemaPage.svelte

```diff
@@ -87,7 +99,12 @@
     </slot>
   </AppSecondaryHeader>
 
-  <TabContainer bind:activeTab {tabs} uniformTabWidth={false}>
+  <TabContainer
+    {activeTab}
+    {tabs}
+    uniformTabWidth={false}
+    on:tabSelected={handleTabSelected}
+  >
```

Comment:
The tabs here should not be buttons with an on click event navigating to a route. They should be anchors, so that users can also right click and open them from in a different browser window/tab.